{{ cookiecutter.project_slug }}/docker/images/api/files/entrypoint.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+python << END
+import sys
+import time
+
+import psycopg
+
+suggest_unrecoverable_after = 30
+start = time.time()
+
+while True:
+    try:
+        con = psycopg.connect(conninfo="${{ '{' }}{{ cookiecutter.__env_prefix }}DATABASE_URL}")
+        con.close()
```

Comment:
Would you like to move closing and break lines in else block?

Reply:
done